templates/partners/ihv-and-oem.html

```diff
@@ -7,50 +7,30 @@
-      </div>
-      <div class="col-4 u-hide--small u-hide--medium u-align--center u-vertically-center">
-        <img src="https://assets.ubuntu.com/v1/0cf81aee-servers-wht.svg" alt="">
-      </div>
+
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-7">
+      <h1>IHV and OEM Programme</h1>
+      <p>Pioneering open source for the modern data centre.</p>
+      <p>For nearly two decades, Canonical and our enterprise partners have provided market&dash;leading solutions for customers seeking alternatives to complex and costly proprietary operating environments. Along the way, we have pioneered open source technologies for modern data centres. Whether you&rsquo;re building physical, virtual, or cloud environments, you can be confident in our tested and certified joint partner solutions.</p>
+      <a href="" class="p-button--positive">Get in touch</a>
+    </div>
+    <div class="col-5 u-hide--small u-hide--medium u-align--center u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/85af7f76-servers-wht-grey.png",
+        alt="",
+        width="433",
+        height="200",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
     </div>
   </div>
 </section>
-<style>
-  .p-strip--suru-top {
-    background-image:
-      linear-gradient(to top right, #fff 0%, #fff 49%, transparent 50%),
-      linear-gradient(to top right, rgba(74, 24, 60, 0) 0%, rgba(74, 24, 60, 0) 49.9%, rgba(74, 24, 60, 0.3) 50%, rgba(74, 24, 60, 0.3) 100%),
-      linear-gradient(to bottom right, rgba(210, 180, 202, 0.05) 0%, rgba(210, 180, 202, 0.05) 49.9%, rgba(210, 180, 202, 0) 50%, rgba(210, 180, 202, 0) 100%),
-      linear-gradient(216deg, #E95420 0%, #5E2750 33%, #2C001E 71%);
-    background-repeat: no-repeat;
-    background-size: 110% 10%, 70% 100%, 100% 40%, 100% 100%;
-    background-position: bottom -0.2px left, top right, top left;
-    padding-bottom: 6rem;
-    padding-top: 3rem;
-  }
-
-  @media (max-width: 772px) {
-    .p-strip--suru-top {
-      background-image:
-        linear-gradient(to top right, #fff 0%, #fff 49%, transparent 50%),
-        linear-gradient(to top right, rgba(74, 24, 60, 0) 0%, rgba(74, 24, 60, 0) 49.9%, rgba(74, 24, 60, 0.3) 50%, rgba(74, 24, 60, 0.3) 100%),
-        linear-gradient(to bottom right, rgba(210, 180, 202, 0.05) 0%, rgba(210, 180, 202, 0.05) 49.9%, rgba(210, 180, 202, 0) 50%, rgba(210, 180, 202, 0) 100%),
-        linear-gradient(216deg, #E95420 0%, #5E2750 33%, #2C001E 71%);
-      background-repeat: no-repeat;
-      background-size: 105% 5%, 70% 100%, 100% 40%, 100% 100%;
-      background-position: bottom -1px left, top right, top left;
-    }
-  }
-</style>
 
-<section class="p-strip is-deep">
+<section class="p-strip has-accent-border--top-left is-deep">
   <div class="u-fixed-width u-align--center">
     <div class="p-logo-section">
       <p class="p-muted-heading u-no-max-width">A selection of our IHV and OEM partners</p>
```

Comment:
This list of logos is a bit different from the one on the doc. Also, chevron after "See all IHV/OEM partners" is missing (won't let me add a comment to that line for some reason)

Reply:
This is another I am waiting on a reply for, because we pull logos from the api on canonical and this is what appears on live.

Reply:
@petesfrench You can edit it here https://partners.ubuntu.com/admincms/partner/?o=-6&programme__id__exact=4, just need whatever logos they want to show to have "published" toggled on and "featured" set to true

Reply:
Thank you @mtruj013, I have decided that as this is external to do it all in one go after the pages are completed